### DISCUSSION TITLE
It possible to publish my Teams app without company to Team Store?

### DISCUSSION THREAD

**Discussion by YuriyMorozyuk95** [microsoft/hack-together-teams#140] [Q&A]
It possible to publish my Teams app without company, and company email to Team Store?
Just use my simple one like Yurii.Moroziuk@hotmail.com

**Comment by waldekmastykarz** [comment]
Please check out our documentation about publishing apps for Teams to the Microsoft Teams store: https://learn.microsoft.com/microsoftteams/platform/concepts/deploy-and-publish/appsource/publish

**Comment by summzhan** [comment]
Why do you want to publish without a company? Most Teams app are intended for collaborate with others in company. If it is for testing purpose, I think you don't need to publish it. You can upload the app package and run it only for yourself. Or share the package to the specific group of people and testing among the group. Hope this is helpful.

**Comment by YuriyMorozyuk95** [comment]
I would like to share app, with my company or with others, mybe will be easier to re-write bot to telegram/facebook etc. But still would like share first app to Teams Store, but I not sure do I want to open company for this thinks, and create bussness email.
